### PR TITLE
feat(accordion2): implement new accordion - INNO-1512

### PR DESCRIPTION
**Pull request by yhuard**
New specs: https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Accordion+amendments

Docs: https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Accordion+ECL+Page+-+final

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 225d8eeaec0e7cdff1d59d216b8727424db130cf

https://deploy-preview-1159--europa-component-library.netlify.com

File: src/systems/ec/implementations/vanilla/packages/ec-component-accordion/ec-component-accordion2.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
+/**
+ * Accordion
+ * @define accordion2; weak
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+// Check if overridden dependencies are already loaded, if needed
+@include check-imports(('ec-component-button'));
```
Comment (emeryro) [inline src/systems/ec/implementations/vanilla/packages/ec-component-accordion/ec-component-accordion2.scss]: we should check ec-component-icon too (same for eu)

**Review by emeryro — CHANGES_REQUESTED** [review]
almost good

File: src/systems/ec/implementations/vanilla/packages/ec-component-accordion/ec-component-accordion2.scss (hunk 1 above)
Comment (yhuard) [reply src/systems/ec/implementations/vanilla/packages/ec-component-accordion/ec-component-accordion2.scss]: fixed

**Review by emeryro — APPROVED** [review]

**Pull request merged**